website/docs/setup/guides/getting-started.md

```diff
@@ -106,22 +100,101 @@ order to print the full event structure:
 <CodeHeader fileName="vector.toml" />
 
 ```diff
+ # Consume data
+ [sources.foo]
+   type = "socket"
+   address = "0.0.0.0:9000"
+   mode = "tcp"
+
+
++# Structure the data
++[transforms.apache_parser]
++  inputs = ["foo"]
++  type = "regex_parser"
++  field = "message"
++  regex = '^(?P<host>[\w\.]+) - (?P<user>[\w]+) (?P<bytes_in>[\d]+) \[(?P<timestamp>.*)\] "(?P<method>[\w]+) (?P<path>.*)" (?P<status>[\d]+) (?P<bytes_out>[\d]+)$'
++
+ # Write the data
+ [sinks.bar]
+-  inputs = ["foo"]
++  inputs = ["apache_parser"]
+   type = "elasticsearch"
+   index = "example-index"
+   host = "http://10.24.32.122:9000"
+```
+
+</TabItem>
+<TabItem value="new_result">
+
+<CodeHeader fileName="vector.toml" />
+
+```toml
+# Consume data
 [sources.foo]
-  type = "stdin"
+  type = "socket"
+  address = "0.0.0.0:9000"
+  mode = "tcp"
+
+# Structure the data
+[transforms.apache_parser]
+  inputs = ["foo"]
+  type = "regex_parser"
+  field = "message"
+  regex = '^(?P<host>[\w\.]+) - (?P<user>[\w]+) (?P<bytes_in>[\d]+) \[(?P<timestamp>.*)\] "(?P<method>[\w]+) (?P<path>.*)" (?P<status>[\d]+) (?P<bytes_out>[\d]+)$'
+
+# Write the data
+[sinks.bar]
+  inputs = ["apache_parser"]
+  type = "elasticsearch"
+  index = "example-index"
+  host = "http://10.24.32.122:9000"
+```
+</TabItem>
+</Tabs>
+
+This regular expression looks great and it probably works, but it's best to be
+sure, right?
+
+## 4. Test it
+
+No one is saying that explosions aren't cool, but you should be doing that in
+your own time and with your own infrastructure.
+
+Vector takes testing seriously and has support for writing
+[unit tests][docs.guides.unit_testing] as part of your config, so let's humor it
+by writing one for our log parser:
```

Comment:
I think the drive here is that testing _should_ be easier. Manual testing would require a local ES instance or actually deploying Vector to production. So I'd rephrase this like:

So how do we know all of this works? We could setup a local Elasticsearch instance, or deploy Vector to production and hope for the best. Vector is different! With Vector you can write simple unit tests directly in your configuration file, just like you would with code. Now you know your Vector config file works without all of the time wasting manual testing. Here's how:

Reply:
Agreed, reworded this section, how do you feel about the new version?